go/consensus/tendermint/apps/keymanager/keymanager.go

```diff
@@ -351,19 +391,34 @@ nextNode:
 		if !isInitialized {
 			panic("the key manager must be initialized")
 		}
+		if secretReplicated {
+			nextRSK = nRSK
+			updatedNodes = append(updatedNodes, n.ID)
+		}
 
 		// If the key manager is not initialized, the first verified node gets to be the source
 		// of truth, every other node will sync off it.
 		if !status.IsInitialized {
 			status.IsInitialized = true
 			status.IsSecure = isSecure
-			status.Checksum = checksum
 		}
 		status.RSK = RSK
-
 		status.Nodes = append(status.Nodes, n.ID)
 	}
 
+	// Accept the proposal if the majority of the nodes have replicated
+	// the proposal for the next master secret.
+	if numNodes := len(status.Nodes); numNodes > 0 && nextChecksum != nil {
+		percent := len(updatedNodes) * 100 / numNodes
+		if percent > minProposalReplicationPercent {
```

Comment:
the consensus app has subtly off-by-one logic, it rejects if percent < 66 when deciding to allow a proposal to be published. `>=` would be consistent here